lib/std/alloc/arena.zig

```diff
@@ -0,0 +1,285 @@
+//! This allocator takes an existing allocator, wraps it, and provides an interface
```

Comment:
I think this file should be named Arena.zig, not arena.zig?

Reply:
yes, almost certainly, will fix!